src/components/cvaa-overlay/cvaa-overlay.js

```diff
@@ -132,7 +134,13 @@ class CVAAOverlay extends Component {
   render(props: any): React$Element<any> {
     props.clearAccessibleChildren();
     return (
-      <Overlay open handleKeyDown={this.props.handleKeyDown} addAccessibleChild={this.props.addAccessibleChild} onClose={props.onClose} type="cvaa">
+      <Overlay
+        open
+        handleKeyDown={this.props.handleKeyDown}
+        addAccessibleChild={this.props.addAccessibleChild}
+        onClose={props.onClose}
+        type="cvaa"
+        label={props.cvvaDialogText}>
```

Comment:
add `{props.settingsText + " " + 'dialog'}`, otherwise the screenreader won’t indicate it is a dialog

"Perhaps we should consider adding translations for this. You can ask Dana or David for assistance. In the meantime, I suppose it's acceptable as it is."